README.md

```diff
@@ -4,7 +4,7 @@ A [musl experiment][].
 
   [musl experiment]: https://github.com/WebAssembly/musl/blob/landing-branch/README.md
 
-The goal of this prototype was to get a WebAssembly libc off the ground. Limited
+The goal of this prototype was to get another WebAssembly libc off the ground. Limited
 dynamic linking (no cross-module function pointers) came out of it *for free*
```

Comment:
It would be useful to contrast here with the existing one. Is this experiment intended to explore an area you think can be improved in the existing one - and if so what? Or is it to get experience from doing a "from-scratch" port?